docs/source/daml/intro/4_Transformations.rst

```diff
@@ -64,7 +64,7 @@ You exercise choices using the ``exercise`` function, which takes a ``ContractId
 
 ``exerciseCmd`` returns a ``Commands r`` where ``r`` is the return
 type specified on the choice, allowing the new ``ContractId Contact``
-to be stored in the variable ``new_contactCid``. Just like for
+to be stored in the variable ``newContactCid``. Just like for
```

Comment:
```suggestion
to be stored in the variable ``newContractCid``. Just like for
```
Maybe needs other changes to sync up the name.

Reply:
It's really contacts: https://github.com/digital-asset/daml/blob/main/docs/source/daml/intro/daml/daml-intro-4/daml/Contact.daml#L68

Reply:
lol I clearly stared at too much daml if I didn’t realize that "contact" is a valid word :)